bundles/org.openhab.binding.salus/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="salus" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Salus Binding</name>
+	<description>
+		This is the binding for Salus, a renowned manufacturer of smart home solutions. This binding allows
+		integration of Salus' wide range devices into your home control system, facilitating bidirectional communication
+		and
+		control.
```

Comment:
```suggestion
		and control.
```

Reply:
FYI this is done by the formatter
![obraz](https://github.com/openhab/openhab-addons/assets/2962846/5afa92a9-29da-48db-b301-b6562af0bd50)

Reply:
https://github.com/openhab/openhab-addons/actions/runs/7434103759/job/20227886902?pr=16065#step:9:222